.clang-format

```diff
@@ -0,0 +1,74 @@
+Language:        Cpp
```

Comment:
We could probably DRY this up and make it a symbolic reference to the clang format in the sundown sub module.